src/components/Question.jsx

```diff
@@ -93,6 +94,27 @@ export default class Question extends React.Component {
     this.props.onChange(this.props.index, newState);
   }
 
+  isDebugged(question) {
+    const startingQuestionId = this.context.options.startingQuestionId;
+
+    if (FormUtils.isRelevant(question)) {
+      return false;
+    }
+    if (this.context.options.debugMode && !FormUtils.isRelevant(question)) {
```

Comment:
```suggestion
    if (this.context.options.debugMode) {
```
`FormUtils.isRelevant(question) == false` here every call ...

Reply:
Done